package.json

```diff
@@ -697,9 +732,13 @@
       "@trezor/connect-web>@trezor/connect>@trezor/utxo-lib>tiny-secp256k1": false,
       "@storybook/test-runner>@swc/core": false,
       "@lavamoat/lavadome-react>@lavamoat/preinstall-always-fail": false,
-      "tsx>esbuild": false,
       "@metamask/eth-trezor-keyring>@trezor/connect-web>@trezor/connect>@trezor/protobuf>protobufjs": false,
-      "firebase>@firebase/firestore>@grpc/proto-loader>protobufjs": false
+      "firebase>@firebase/firestore>@grpc/proto-loader>protobufjs": false,
+      "tsx>esbuild": true,
```

Comment:
I turned this on, because I think we want to allow esbuild to build itself if needed?